contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Why are we redesigning PostHog?
+date: 2023-11-27
+author: ["cory-watilo"]
+rootpage: /blog
+featuredImage: ../images/blog/why-redesign/preview-3000.jpg
+featuredImageType: full
+category: PostHog news
+tags:
+  - Product updates
+---
+
+A year ago, [James](/james) and I sat down in a coffee shop in Austin, Texas and had a discussion about what the future of PostHog could look like.
+
+Our current UI has served its purpose, but as we grow as a company, the possibilities of what can be achieved in PostHog have grown – as have our ambitions.
+
+We’re launching a reskin this month as the first step in a series of many milestones around the experience of using PostHog. I’d like to share some of the reasoning behind our changes, and the longer-term vision we’re aiming to realize in the future.
```

Comment:
```suggestion
We’re launching a redesign this month as the first step in a series of many milestones around the experience of using PostHog. I’d like to share some of the reasoning behind our changes, and the longer-term vision we’re aiming to realize in the future.
```

Can we be consistent is referring to this as a redesign? Bettter than Redesign/Reskin/Refresh/Rebuild.